src/controllers/softtakeover.cpp

```diff
@@ -102,32 +102,74 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
 
     qint64 currentTime = Time::elapsedMsecs();
     // We will get a sudden jump if we don't ignore the first value.
-    if (m_time == 0) {
+    if (m_time == -1) {
         ignore = true;
         // Change the stored time (but keep it far away from the current time)
         //  so this block doesn't run again.
-        m_time = 1;
-//         qDebug() << "SoftTakeover::ignore: ignoring the first value"
-//                  << newParameter;
-    } else if ((currentTime - m_time) > SUBSEQUENT_VALUE_OVERRIDE_TIME_MILLIS) {
-        // don't ignore value if a previous one was not ignored in time
+        m_time = -2;    // Tests like to use 0
```

Comment:
-2 is not enough for the fast tests use -kSubsequentValueOverrideTimeMsecs to be sure.